:q:q

```diff
@@ -0,0 +1,292 @@
+[33mcommit e27aa850714f27f1d00881b9fbb38121fcc9a8f9[m[33m ([m[1;36mHEAD -> [m[1;32mfeature/Yelb-app[m[33m)[m
```

Comment:
What is this? unsuccessful attempt to get out of vim?

Reply:
Sorry this is a commit history. Removed it now